CHANGELOG.md

```diff
@@ -20,19 +27,19 @@
 - new module `stdlib_linalg`
   - new procedures `diag`, `eye` and `trace`
     [#170](https://github.com/fortran-lang/stdlib/pull/170)
-  - new procedures `linspace` and `logspace`
-    [#420](https://github.com/fortran-lang/stdlib/pull/420)
-  - new procedure `outer_product`
-    [#432](https://github.com/fortran-lang/stdlib/pull/432)
-  - new procedure `arange`
-    [#480](https://github.com/fortran-lang/stdlib/pull/480)
 - new module `stdlib_logger`
   - new derived type: `logger_type`
     [#228](https://github.com/fortran-lang/stdlib/pull/228)
     [#261](https://github.com/fortran-lang/stdlib/pull/261)
 - new module `stdlib_math`
   - new procedure `clip`
     [#355](https://github.com/fortran-lang/stdlib/pull/355)
+  - new procedures `linspace` and `logspace`
+    [#420](https://github.com/fortran-lang/stdlib/pull/420)
+  - new procedure `outer_product`
+    [#432](https://github.com/fortran-lang/stdlib/pull/432)
+  - new procedure `arange`
+    [#480](https://github.com/fortran-lang/stdlib/pull/480)
```

Comment:
Two days ago, the `gcd` function was also added to `stdlib_math`. (`gcd`, see https://github.com/fortran-lang/stdlib/pull/539)

Now `stdlib_math` has: `clip, linspace, logspace, arange, gcd`.